IPSec VPN edits (#1164)
saucelabs/sauce-docs: docs/secure-connections/ipsec-vpn.md

* How to run tests using Sauce IPSec Proxy

### What You'll Need

<p> <Highlight color="#013a70">Enterprise Plans Only</Highlight> </p>

* Sauce IPSec Proxy is an enterprise-only feature that must be configured by our Support Team prior to use. Contact your Sauce Labs Sales Engineer or Customer Success Manager to learn more about how this solution can meet your business needs.
* Authorization to use your organization's Sauce IPSec Proxy tunnel. You can verify this by going to the **Tunnels** page, where you should see the tunnel name displayed. If you don't see it, contact the organization admin for your Sauce Labs account to request access.

## Sauce IPSec Proxy Setup

As part of our initial Sauce IPSec Proxy tunnel setup and configuration on your network, we'll ask your organization to choose from two sharing settings:

* **Restricted**: Only organization admins can utilize Sauce IPSec Proxy tunnels
* **Organization-wide**: All users in your organization can utilize Sauce IPSec Proxy tunnels

If you're an organization admin who would like to change your sharing settings, please reach out to our Support Team.

### Public Real Devices

> To run tests on public real devices in the Sauce Labs cloud using Sauce IPSec Proxy, organization admins must switch on **Enable Sauce Connect Proxy/IPSec VPN for Public Cloud Devices**, a security setting that is disabled by default. See [Security Settings](/basics/acct-team-mgmt/org-settings) for more information.

Reviewer: ```suggestion
To run tests on public real devices in the Sauce Labs cloud using Sauce IPSec Proxy, organization admins must set the **Enable Sauce Connect Proxy/IPSec VPN for Public Cloud Devices** option to **Enabled**. See [Security Settings](/basics/acct-team-mgmt/org-settings) for more information.
```
